seqr/views/utils/permissions_utils.py

```diff
@@ -44,6 +53,39 @@ def test_user(user):
 def _current_policies_required(view_func, policy_url=API_POLICY_REQUIRED_URL):
     return user_passes_test(_has_current_policies, login_url=policy_url)(view_func)
 
+
+class ProgrammaticAccess:
```

Comment:
Can we find a better name than programmatic access, particularly to make the distinction with the existing `login_and_policies_required` more clear? How about `service_account_required` together with `ServiceAccountAccess` or similar?

Reply:
Robot account? Automation account? ;)

Reply:
Renamed "programmatic" to "service account" :)

Reply:
What about token_access?